cmake/libappimage.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${prefix}/lib
```

Comment:
Instead of `lib` you might want to use `@CMAKE_INSTALL_LIBDIR@`, so its the proper lib, lib64 or debian-variant of the system.

Reply:
Agreed, done.